roles/nginxplus/files/conf/http/templates/pudl_proxy_pass.conf

```diff
@@ -0,0 +1,307 @@
+# arks redirects
+    rewrite ^/objects/(.*)$ /objects/$1 redirect;
+    location /objects/ {
+        proxy_pass https://arks.princeton.edu/ark/88435/;
+    }
+
+    rewrite ^/sheetreader.php?obj=(.*)$ /sheetreader/$1 redirect;
+    location /sheetreader/ {
+        proxy_pass https://arks.princeton.edu/ark/88435/;
+    }
+
+# dpul redirects
+#
+    rewrite ^/search.php(.*)$ /search.php$1 redirect;
+    location /search.php {
+        proxy_pass https://dpul.princeton.edu/search.php;
+    }
+
+    rewrite ^/collections.php(.*)$ /collections.php$1 redirect;
+    location /collections.php {
+        proxy_pass https://dpul.princeton.edu/;
+    }
+
+    rewrite ^/about.php(.*)$ /about.php$1 redirect;
+    location /about.php {
+        proxy_pass https://dpul.princeton.edu/;
+    }
+
+    rewrite ^/collections/eal(.*)$ /eal$1 redirect;
+    location /eal {
+        proxy_pass https://dpul.princeton.edu/eastasian;
+    }
+
+    rewrite ^/collections/garden(.*)$ /garden$1 redirect;
+    location /garden {
+        proxy_pass https://dpul.princeton.edu/marquand-library;
+    }
+
+    rewrite ^/collections/papyri(.*)$ /papyri$1 redirect;
+    location /papyri {
+        proxy_pass https://dpul.princeton.edu/papyri;
+    }
+
+    rewrite ^/collections/paris(.*)$ /paris$1 redirect;
+    location /paris {
+        proxy_pass https://dpul.princeton.edu/marquand-library;
+    }
+
+    rewrite ^/collections/scheide(.*)$ /scheide$1 redirect;
+    location /scheide {
+        proxy_pass https://dpul.princeton.edu/scheide;
+    }
+
+    rewrite ^/collections/versailles(.*)$ /versailles$1 redirect;
```

Comment:
Will this handle both `/collections/versailles2` and `/collections/versailles3`?

Reply:
it should. Only way to test is after the fact. Unfortunately. :-(